packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -98,6 +98,11 @@ export class AtomicSearchInterface {
    */
   @Prop() public reflectStateInUrl = true;
 
+  /**
+   * Whether analytics should be enabled.
+   */
+  @Prop({reflect: true}) public analyticsEnabled = true;
```

Comment:
I think this prop can be shortened to `analytics` so that it appears in the html as:

```
<atomic-search-interface analytics="false"/>
```

That said, @erocheleau made a great point in the JSUI PR for using the negative form to align with the html spec. I think it could be an alternative in this case,

```
<atomic-search-interface disable-analytics/>

<atomic-search-interface no-analytics/>
```

Reply:
I'll go with `disable-analytics` !